internal/provider/network/schema_network.go

```diff
@@ -39,12 +39,14 @@ func Schema() map[string]*schema.Schema {
 		constants.FieldNetworkRouteCIDR: {
 			Type:          schema.TypeString,
 			Optional:      true,
+			Computed:      true,
```

Comment:
IIRC, both CIDR and gateway can be configured by the users, so computed may not be a valid solution.
https://www.terraform.io/plugin/sdkv2/schemas/schema-behaviors#computed

Reply:
Users can set value to a Computed field

Reply:
Does it mean the `computed` definition can help to merge the output after `tf apply` and it is also configurable if needed?

Reply:
Yes